wp-content/themes/pub/wporg-learn-2024/src/style/_print.scss

```diff
@@ -0,0 +1,47 @@
+@media print {
+	// Hide stuff that's not relevant on print media
+	.wp-block-wporg-local-navigation-bar,
+	.wp-block-wporg-site-breadcrumbs,
+	.wporg-learn-sidebar,
+	footer.wp-block-template-part {
```

Comment:
Looks like this does the trick

```suggestion
	footer.wp-block-template-part,
	.global-header {
```

Reply:
Ah yeah, it looks nice. 3c6acfb

![image](https://github.com/user-attachments/assets/13e64ab0-196a-4d9b-bded-6b409fdd0e68)